Description: Change directory form usr/local to usr/share
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 libvbts (0.0.1-1) unstable; urgency=low
 .
   * source package automatically created by stdeb 0.6.0+git
Author: Kurtis Heimerl <kheimerl@cs.berkeley.edu>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- libvbts-0.0.1.orig/setup.py
+++ libvbts-0.0.1/setup.py
@@ -11,7 +11,7 @@ setup(name="libvbts",
       scripts=[],
       install_requires=['smspdu'],
       data_files=[
-        ("/usr/local/freeswitch/scripts",['freeswitch/VBTS_Parse_SMS.py',
+        ("/usr/share/freeswitch/scripts",['freeswitch/VBTS_Parse_SMS.py',
                                           'freeswitch/VBTS_New_User.py',
                                           'freeswitch/VBTS_Send_SMS.py',
                                           'freeswitch/VBTS_Send_Empty_SMS.py',
